docs/guidance/model/milestones.md

```diff
@@ -0,0 +1,141 @@
+
+The `dueDate`, `dateMet`, `dateModified` and [`status`](../../../schema/codelists/#milestone-status) are used to track the lifecycle of the milestone.
+
+## Worked examples
+
+The following worked examples will show how to fill the milestone block depending on its nature.
+
+### Planning and tender milestones
+
+The planning milestone block should be used only to disclose information about the planning process, such as the 
+budget approval, key studies etc. If during the planning stage you have information about tender process milestones, then you
+should populated the tender/milestones block instead.
+
+In the example below:
+
+* The process starts at the planning stage, and the information about when the budget plan is estimated to be ready 
+is available. We create a planning release including a milestone with type set to 'preProcurement', `dueDate` set to the estimated date, and `status` set to 'scheduled'. 
+The publisher also knows when the tender notice is estimated to take place. This information is part of the **tender** process.  A `milestone` is created in the `tender/milestones` block with type set to 'preProcurement' and the relevant date set in the `dueDate` field. The tender must have a `status` set to 'planned'.
+* The process goes forward and enters the tender stage. We publish a tender release 
+with `tender/status` set to 'active',  set the relevant date in the milestone's `dateMet` field, and set its status to 'met'.
+* To explore differences between the planned and actual date of the tender milestone, users can then compare the values of `tender/milestones/dueDate` and `tender/milestones/dateMet` in a single (compiled) release.
+
+```eval_rst
+
+.. jsoninclude:: ../../examples/milestones/planning-tender-milestones.json
+   :jsonpointer: 
+   :expand: releases, planning, milestones, tender, milestones
+   :title: planning
+
+.. jsoninclude:: ../../examples/milestones/planning-tender-milestones-2.json
+   :jsonpointer: 
+   :expand: releases, planning, milestones, tender, milestones
+   :title: tender
+
+```
+
+### Contract implementation milestones
+
+#### 1. Project data
+
+This worked example shows how milestones for project commencement and project completion should be updated in OCDS.
+
+The example includes three updates to a single contracting process (ocds-213czf-000-00001):
+
+* A contract is awarded to a user committee with milestones for project commencement and project completion.
+* The project starts later than expected. The project commencement milestone is updated with the actual start date and the project completion milestone `status` is set to 'notMet'.
+* The project is completed earlier than expected. The project completion milestone is updated with the actual completion date.
+
+The OCDS releases for each update are shown below. 
```

Comment:
Copy-edit for clarity

```suggestion
The example below includes 3 OCDS releases from a contracting process for the construction of a cycle lane:

* An award release which includes the scheduled start and end dates for the project.
* An implementation update release with the actual start date of the project
* An implementation update release with the actual end date of the project

In the award release:

* The scheduled start and end dates for the project are represented using milestones in `contracts/implementation/milestones` with `.type` set to 'delivery' because they related to the delivery of the contract. The dates are provided in `.dueDate` and `.status` is set to 'scheduled'.
* The publisher has defined their own values for the `.code` field so they can filter and compare start and end date milestones across different contracts.

The first implementation update is published after the project starts but before it completes:

* In the project commencement milestone, `.dateMet` is set to the actual start date and `.status` is set to 'met'. `.dateModified` is set to the date the milestone was updated.
* In the project completion milestone, `.status` is set to 'notMet' since the project is not yet complete and `.dateModified` is set to the date the milestone was updated.

Users can compare the project commencement milestone's `.dueDate` and `.dateMet` fields to determine if the project started on time. Users can also compare the project completion milestone's `.dueDate` and `.dateModified` fields to determine whether the `.status` has been updated since the scheduled completion date.

The second implementation update is published after the project completes:

* In the project completion milestone, `.dateMet` is set to the actual completion date for the project and `.status` is set to 'met'.
```